exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Showroom.java

```diff
@@ -0,0 +1,14 @@
+package com.cbfacademy.cars;
+
+public class Showroom {
+
+     String make; 
+     String model; 
+     int year;
+
+    public Showroom(String make, String model, int year){
```

Comment:
Your Showroom class should be able to create various `Car` objects, so these parameters wouldn't be necessary